src/main/resources/static/views/interest/Add_Interest.html

```diff
@@ -0,0 +1,69 @@
+<div class="container">
+    <div class="row">
+        <div ng-show="error">
+            <div class="alert alert-danger">There was an error: {{error}}</div>
+        </div>
+        <div class="col-xs-12 col-sm-10 col-md-8 col-lg-8 col-lg-offset-2">
+            <div class="panel-card panel-success-card">
+                <div class="text-center" style="margin-top: 6%"><h3>{{!interest.id ? 'Create' :'Edit'}} investment
+                    interest</h3>
+                </div>
+                <div class="panel-body">
+                    <div class="col-xs-12">
+                        <form name="interestForm" id="interestForm" role="form" ng-submit="submit()">
+                            <input type="hidden" ng-model="interest.id">
+                            <div class="form-group">
+                                <label for="goal" class="col-md-4 control-label">Goal</label>
+                                <div class="col-xs-12 col-md-8 col-lg-8" style="margin-bottom: 25px">
+                                    <input type="text" ng-model="interest.investmentGoal"
+                                           class="form-control input-md"
+                                           id="goal" required>
+                                </div>
+                            </div>
+                            <div class="form-group">
+                                <label for="description" class="col-md-4 control-label">Description</label>
+                                <div class="col-xs-12 col-md-8 col-lg-8" style="margin-bottom: 25px">
+                                    <textarea style="resize: none" rows="7" ng-model="interest.description"
+                                              class="form-control input-md" id="description" required></textarea>
+                                </div>
+                            </div>
+                            <div class="form-group">
+                                <label for="amount" class="col-md-4 control-label">Investment</label>
+                                <div class="col-xs-12 col-md-8 col-lg-8" style="margin-bottom: 25px">
+                                    <input type="number" ng-model="interest.amountInvestment"
+                                           max="{{maxAmount}}"
+                                           class="form-control input-md"
+                                           id="amount" name="amount" required>
+                                    <!--<span class="form-validation" ng-show="interestForm.amount.$error.pattern">Not more than 8 numbers</span>-->

```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this commented out code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=Web%3AAvoidCommentedOutCodeCheck)